orbit/pkg/table/extension.go

```diff
@@ -133,7 +134,8 @@ func OrbitDefaultTables() []osquery.OsqueryPlugin {
 		table.NewPlugin("sntp_request", sntp_request.Columns(), sntp_request.GenerateFunc),
 
 		// Kolide extensions.
-
+		zfs.ZfsPropertiesPlugin(serverClient, kolideLogger),   // table name is "kolide_zfs"
```

Comment:
Let's pass `nil` here.

It seems many of the tables do not use such argument.

(I will remove `serverClient` on another PR because if any of the tables need it we must pass something valid.)